spec/lib/importers/rss_feed_data_spec.rb

```diff
@@ -31,23 +35,23 @@
         stub_request(:get, rss_feed_url.url).to_return({ status: 200, body: rss_feed_content })
         allow(UrlStatusCodeFetcher).to receive(:fetch) do |arg|
           status_code =
-              case arg
-                when 'http://www.whitehouse.gov/blog/2011/09/26/famine-horn-africa-be-part-solution'
-                  '404 Not Found'
-                else
-                  '200 OK'
-              end
+            case arg
+            when 'http://www.whitehouse.gov/blog/2011/09/26/famine-horn-africa-be-part-solution'
+              '404 Not Found'
+            else
+              '200 OK'
+            end
           { arg => status_code }
         end
         rss_feed_url.news_items.destroy_all
         described_class.new(rss_feed_url).import
       end
 
-      it 'should populate and index just the news items that are valid' do
+      it 'populates and index just the news items that are valid' do
```

Comment:
Should be "populates and indexes". 

It's a symptom of a spec that's testing too many things, but unfortunately that's the case for a lot of our older specs. I'm going to create a ticket for one of the new folks to disable auto-correction for that cop, as it has tripped us up in various ways in the past.

See also https://github.com/rubocop/rubocop-rspec/issues/1179